internal/security/setup/option/optionHelpers.go

```diff
@@ -178,6 +178,25 @@ func checkIfFileExists(fileName string) bool {
 	return !fileInfo.IsDir()
 }
 
+func checkDirExistsAndIsWritable(path string) (exists bool, isWritable bool) {
+	exists = false
+	isWritable = false
+
+	fileInfo, statErr := os.Stat(path)
+	if os.IsNotExist(statErr) {
+		return
+	}
+
+	exists = fileInfo.IsDir()
+
+	fileMode := int(fileInfo.Mode())
+	writableMask := int(0200)
```

Comment:
personally, I prefer this as `(1 << 7)` (with appropriate casting, etc.) as I always forget `0200` is _octal_...

Reply:
we could define that as a constant